### PR TITLE
Remove warning about 1mb+ Worker size

### PR DESCRIPTION
**Pull request by irvinebroque**

### CODE REVIEW

**Comment by irvinebroque** [comment]
@petebacondarwin — assume there might be a bit more here to get this over the line, but just opening to start the ball rolling. I see people trip up on this, made sense with previous constraints but no longer.

File: packages/wrangler/src/deployment-bundle/bundle-reporter.ts [hunk 1]
```diff
@@ -40,12 +40,6 @@ export async function printBundleSize(
 				: chalk.green(bundleReport);
 
 	logger.log(`Total Upload: ${colorizedReport}`);
-
-	if (gzipSize > ALLOWED_INITIAL_MAX && !process.env.NO_SCRIPT_SIZE_WARNING) {
```
Comment (CarmenPopoviciu) [inline packages/wrangler/src/deployment-bundle/bundle-reporter.ts:44]: @irvinebroque I can take over this PR as on-call, but I need to understand a bit better what it is that we're trying to do. Is it just the copy that became irrelevant here? The 1MB Worker size is retained in `ALLOWED_INITIAL_MAX`, which is used to calculate the colorized report just above (see `logger.log(`Total Upload: ${colorizedReport}`);`). Is that value of 1MB outdated as well? If yes, do we have a new max?

Comment (CarmenPopoviciu) [reply packages/wrangler/src/deployment-bundle/bundle-reporter.ts:44]: perhaps there's an internal ticket/external GH issue you can point me to for more details? 🙏

**Comment by petebacondarwin** [comment]
Closing in favour of https://github.com/cloudflare/workers-sdk/pull/6381 which contains this.

**Pull request closed without merging**